.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+<!-- 🚨 Please Do Not skip any instructions and information mentioned below as they are all required and essential to investigate the issue. Issues with missing information may be closed without investigation 🚨 -->
+
+## Describe the issue
+A clear and concise description of what the issue is.
```

Comment:
Maybe we provide examples of types of issues we expect here? e.g.

- Typo or confusion around a particular wiki article
- Suggestion for a topic to cover within the wiki
- Proposal for a change to overall project governance